log/format.go

```diff
@@ -14,10 +14,11 @@ import (
 )
 
 const (
-	timeFormat     = "2006-01-02T15:04:05.999999999-0700"
-	termTimeFormat = "01-02|15:04:05"
-	floatFormat    = 'f'
-	termMsgJust    = 40
+	timeFormat        = "2006-01-02T15:04:05-0700"
+	termTimeFormat    = "01-02|15:04:05"
+	newTermTimeFormat = "2006-01-02T15:04:05.999999999-0700"
```

Comment:
Can we rename this to something more semantically descriptive? like `nanoTermTimeFormat`

also, just want to verify that timeFormat and termTimeFormat still being used

Reply:
timeFormat is used somewhere else. `nanoTermTimeFormat` replaces `termTimeFormat`. The IDE doesn't find any usage of it.

Reply:
gotcha. then I'd say use the name `termTimeFormat` to represent the new time format with nano-seconds.